files/image_config/rsyslog/rsyslog.conf.j2

```diff
@@ -64,42 +64,9 @@ template(name="WelfRemoteFormat" type="string" string="%TIMESTAMP% id=firewall t
 :::date-year%-%timereported:::date-month%-%timereported:::date-day% %timereported:::date-hour%:%timereported:::date-minute%:%timereported\
 :::date-second%\" fw=\"{{ fw_name }}\" pri=%syslogpriority% msg=\"%syslogtag%%msg:::sp-if-no-1st-sp%%msg:::drop-last-lf%\"\n")
 
-#
-# Set the default permissions for all log files.
-#
-$FileOwner root
-$FileGroup adm
-$FileCreateMode 0640
-$DirCreateMode 0755
-$Umask 0022
-
-#
-# Where to place spool and state files
-#
-$WorkDirectory /var/spool/rsyslog
```

Comment:
Suggest keep permission and working directory config before remote syslog config.

According to doc, syslog may buffer massive data on disk with spool, not sure if move these after the remote syslog config will have impact:
https://www.rsyslog.com/doc/tutorials/high_database_rate.html

Reply:
If this is from day one then this is OK